help_to_heat/frontdoor/interface.py

```diff
@@ -221,9 +219,8 @@ def find_addresses(self, building_name_or_number, postcode):
 
     @with_schema(load=GetAddressSchema, dump=FullAddressSchema)
     def get_address(self, uprn):
-        api = osdatahub.PlacesAPI(settings.OS_API_KEY)
-        api_results = api.uprn(int(uprn), dataset="LPI")["features"]
-        address = api_results[0]["properties"]["ADDRESS"]
+        api_results = MockOSApi(settings.OS_API_KEY).get_by_uprn(int(uprn), "LPI")["results"]
```

Comment:
Before, it was using `OSApi` in `.os_api` to get postcode results, but using `osdatahub.PlacesAPI(settings.OS_API_KEY)` to get uprn results. I changed it to use the same api.